nilrt_snac/requirements.txt

```diff
@@ -0,0 +1,2 @@
+mypy 
+ni-python-styleguide
```

Comment:
You've dropped the linting target entirely; right? Do you still need these modules?